frontend/common/OfflineHTMLExport.js

```diff
@@ -1,27 +1,60 @@
 const CDNified = (version, file) => `https://cdn.jsdelivr.net/gh/fonsp/Pluto.jl@${version.substr(1)}/frontend/${file}`
 
 export const offline_html = ({ pluto_version, body, head }) => {
+    
     Array.from(body.querySelectorAll(".CodeMirror-sizer")).forEach((s) => (s.style.minHeight = "24px"))
 
-    return `<!DOCTYPE html>
-    <html lang="en">
-    <head>
-        <meta name="viewport" content="width=device-width" />
-        <title>⚡ Pluto.jl ⚡</title>
-        <meta charset="utf-8" />
+    var blob_to_base64_promises = [];
+    Array.from(body.querySelectorAll("img")).forEach((img) => {
+        if (img.src.match(/^blob:/)) {            
+            blob_to_base64_promises.push(new Promise((resolve) => {
+                var xhr = new XMLHttpRequest();
```

Comment:
I found this SO answer, looks like we can use `fetch` instead - that would be easier on the eyes

https://stackoverflow.com/questions/11876175/how-to-get-a-file-or-blob-from-an-object-url

I'll try this myself